stable/velero/README.md

```diff
@@ -0,0 +1,103 @@
+# Velero-server
+
+This helm chart installs Velero version v0.11.0
+https://github.com/heptio/velero/tree/v0.11.0
+
+## Upgrading to v0.10
```

Comment:
This should be v0.11.0, and instructions for upgrading from v0.10.x to v0.11.0 can be found at https://heptio.github.io/velero/v0.11.0/migrating-to-velero

Reply:
Doh, I see those are further down below.

IMO, the v0.10 section instructions should be removed.

Reply:
Done.